lib/features/reps_core/reps_core.module

```diff
@@ -843,3 +843,14 @@ function reps_core_form_page_node_form_alter(&$form, &$form_state, $form_id) {
     $form['body']['#weight'] = -2;
   }
 }
+
+/**
+ * Implements hook_views_pre_render().
+ */
+function reps_core_views_pre_render(&$view) {
+  if ($view->tag == 'nodequeue') {
```

Comment:
using a single if is more readable for me
if ($view->tag == 'nodequeue' && user_is_anonymous())
and can you also add a reference to the ticket in the comment
// REPR-1730 deny access to nodequeue for anonymous
by this way this is possible to understand where this code comes from if needed in the future...